lib/nerves_hub_link/extensions/health/default_report.ex

```diff
@@ -59,6 +61,21 @@ defmodule NervesHubLink.Extensions.Health.DefaultReport do
     |> Enum.reduce(%{}, &Map.merge/2)
   end
 
+  defp get_metric_sets() do
+    Application.get_env(:nerves_hub_link, :health, [])
+    |> Keyword.get(:metric_sets, @default_metric_sets)
+    |> Enum.map(fn metric_set ->
+      if metric_set in [Default, Defaults] do
```

Comment:
Not sure how I feel about this. `Default` and `Defaults` aren't real modules and is effectively the same as `:defaults`, except maybe more misleading since someone might go searching for a non-existent module.

I would vote for not using it and require being explicit. Have the defaults be used if no `:metric_sets` seems appropriate as one could then explicitly set it to `nil` or `[]` to skip those metrics. If we _need_ to have an easy defaults key, then I think I need to relent and go with `:defaults` to prevent someone needlessly digging for a non-existent module

Reply:
I've switched this to `:defaults` and `:default`. I think this is useful to have.